packages/transformers/vue/src/VueTransformer.js

```diff
@@ -225,7 +225,16 @@ async function processPipeline({
       }
       let content = template.content;
       if (template.lang && !['htm', 'html'].includes(template.lang)) {
+        let options = {};
         let preprocessor = consolidate[template.lang];
+        // Pug doctype fix (fixes #7756)
+        switch (template.lang) {
+          case 'pug':
+            options.doctype = 'html';
+            break;
+          default:
+            break;
```

Comment:
Not a big issue but these lines aren't really necessary.

Reply:
True, I can remove them